.circleci/config.yml

```diff
@@ -294,19 +401,19 @@ workflows:
           name: e2e-test RollingRestart
           test_name: RollingRestart
           requires:
-            - unit-test
+            - unit-test-casskop
           filters:
             tags:
               only: /.*/
             branches:
               # Forked pull requests have CIRCLE_BRANCH set to pull/XXX
-              ignore: /pull\/[0-9]+/
+              ignore: /pull\/[0-9w]+/
```

Comment:
@erdrix, not sure that's what you want here. Are you saying we have branches that have digits and `w`s only ?

Reply:
No, it was a mistake !